ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/util/DebugDataDumper.java

```diff
@@ -41,12 +41,9 @@ public class DebugDataDumper {
   private boolean enabled;
   private final Path directory;
 
-  public DebugDataDumper(final Path directory, final boolean enabled) {
-    this.enabled = enabled;
+  public DebugDataDumper(final Path directory) {
+    this.enabled = true;
```

Comment:
hard coded now, lets just fix the code. it doesn't need to exist.

Reply:
This is still used as part of the circuit breaker so we should leave it.